### PR TITLE
Adds ginger beer drinking glass sprite, ginger beer bottles to applicable vendors

### PR DESCRIPTION
**Pull request by toasterroboto**
## About The Pull Request

Adds an animated sprite for ginger beer in a drinking glass and adds a bottle that holds ginger beer, found in the cola machines and boozeomats around the station. Also changes the description of ginger beer.

## Why It's Good For The Game

Ginger beer didn't have a sprite once poured into a glass previously. This fixes that and adds a ginger beer bottle to the boozeomat and cola machines.

## Changelog
:cl:
add: Ginger beer with accompanying bottle to boozeomats and cola machines.
fix: Ginger beer now has a sprite when poured into a drinking glass.
tweak: Ginger beer now has a description that isn't 'mmmm ginger'.
/:cl:

### CODE REVIEW

**Comment by TheFakeElon** [comment]
🙏

**Review by TheFakeElon — APPROVED** [review]

**Comment by TheFakeElon** [comment]
Just remove the beestation DME include change

**Comment by toasterroboto** [comment]
> 
> 
> Just remove the beestation DME include change

done

**Comment by TheFakeElon** [comment]
awesome

File: code/modules/vending/cola.dm [hunk 1]
```diff
@@ -12,6 +12,7 @@
 					/obj/item/reagent_containers/food/drinks/soda_cans/starkist = 10,
 					/obj/item/reagent_containers/food/drinks/soda_cans/space_up = 10,
 					/obj/item/reagent_containers/food/drinks/soda_cans/pwr_game = 10,
+					/obj/item/reagent_containers/food/drinks/bottle/gingerbeer = 10,
```
Comment (TheFakeElon) [inline code/modules/vending/cola.dm:15]:
```suggestion
					/obj/item/reagent_containers/food/drinks/bottle/gingerbeer = 10, austation -- adds ginger beer bottles
```

**Pull request merged**